assets/src/scss/partials/country-selector-new.scss

```diff
@@ -0,0 +1,6 @@
+@import "../base/mixins";
+@import "../base/variables";
+@import "../base/colors";
+@import "../base/fonts";
+
+//@import "../layout/country-selector";
```

Comment:
I for now commented this import, you can choose whether you use it, or move the new code directly in this file.